dftimewolf/lib/collectors/grr_hunt.py

```diff
@@ -247,20 +259,34 @@ def _extract_hunt_results(self, output_file_path):
     # Extract items from archive by host for processing
     collection_paths = []
     clients = set()
+    client_id_fqdn = {}
```

Comment:
Please rename this to something more explanatory - fqdn_by_client_id or client_id_to_fqdn or similar.

Reply:
Done.